Automattic/jetpack
# Jetpack: register new VideoPress block via pkg

This PR registers the new VideoPress block in the Jetpack plugin using the VideoPress package.

Fixes #

#### Changes proposed in this Pull Request:

* Jetpack: register new VideoPress block via package

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






Check you see the VideoPress video block available only under [beta conditions](https://github.com/Automattic/jetpack/tree/trunk/projects/plugins/jetpack/extensions#beta-extensions).

#### Testing Locally

* When `JETPACK_BETA_BLOCKS` is true, you should be able to see the new VideoPress video block.
* Otherwise, it should be not available.
 
#### Testing in JN sites

* Create a JN with Jetpack Beta 
* Create a new post
* Try to add the VideoPress block. Confirm it is not available.
<img width="367" alt="image" src="https://user-images.githubusercontent.com/77539/186421694-3f648fbb-daa4-417d-b38b-4f942191b1b6.png">

* Go to Jetpack constants
* Enable beta extensions
* <img width="783" alt="Screen Shot 2022-08-24 at 09 42 59" src="https://user-images.githubusercontent.com/77539/186421471-80fc6433-8c84-4502-a64b-745c16fac1d1.png">

* Go to edit/create a new post
* Confirm you can add the new VideoPress block

<img width="362" alt="Screen Shot 2022-08-24 at 09 43 41" src="https://user-images.githubusercontent.com/77539/186421434-cc66a20a-c92c-461e-aa3a-fc1aff914006.png">



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1202777791570480
